BagTrackTests/BagsTableViewControllerTests.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  BagsTableViewControllerTests.swift
+//  BagTrackTests
+//
+//  Created by Robin Kipp on 16.11.17.
+//  Copyright © 2017 Robin Kipp. All rights reserved.
+//
+
+import XCTest
+import CoreLocation
+@testable import BagTrack
+
+class BagsTableViewControllerTests: XCTestCase {
+
+    var bundle:Bundle!
```

Comment:
Instead of creating and tearing down all of these instance variables each time, just initialize them when you declare them. Your test case class is re-instantiated for each test method that is run, so you don't have to worry about state being shared between invocations.